packages/rocketchat-i18n/i18n/pt-BR.i18n.json

```diff
@@ -2740,6 +2740,7 @@
   "Remove_as_leader": "Remova como líder",
   "Remove_as_moderator": "Remover como moderador",
   "Remove_as_owner": "Remover de proprietário",
+  "Remove_Channel_Links": "Remover relação entre canais",
```

Comment:
since you added this one, can you please add a translation for `SlackBridge_Remove_Channel_Links_Description` as well? thx